src/ATen/native/xpu/sycl/SortingKernels.h

```diff
@@ -315,6 +316,13 @@ void segmented_radix_sort_pairs_downsweep_kernel(
 
 // ======================= large sort =======================
 
+template <typename scalar_t>
+struct InternalCopyFunctor {
```

Comment:
What's the `Internal` mean? Internal for sort? Or naming with `ABBufferCopyFunctor`.